src/pages/home/report/ReportActionCompose.js

```diff
@@ -742,6 +877,30 @@ class ReportActionCompose extends React.Component {
                     <ExceededCommentLength comment={this.comment} onExceededMaxCommentLength={this.setExceededMaxCommentLength} />
                 </View>
                 {this.state.isDraggingOver && <ReportDropUI />}
+                {!_.isEmpty(this.state.suggestedEmojis) && this.state.shouldShowSuggestionMenu && (
+                    <ArrowKeyFocusManager
+                        focusedIndex={this.state.highlightedEmojiIndex}
+                        maxIndex={getMaxArrowIndex(this.state.suggestedEmojis.length, this.state.isEmojiPickerLarge)}
+                        shouldExcludeTextAreaNodes={false}
+                        onFocusedIndexChanged={index => this.setState({highlightedEmojiIndex: index})}
+                    >
+                        <EmojiSuggestions
+                            onClose={() => this.setState({suggestedEmojis: []})}
+                            highlightedEmojiIndex={this.state.highlightedEmojiIndex}
+                            emojis={this.state.suggestedEmojis}
+                            comment={this.state.value}
+                            updateComment={newComment => this.setState({value: newComment})}
+                            colonIndex={this.state.colonIndex}
+                            prefix={this.state.value.slice(this.state.colonIndex + 1).split(' ')[0]}
```

Comment:
This logic caused this issue https://github.com/Expensify/App/issues/16626 
So that I suggest we should replace it by 
```
prefix={this.state.value.slice(this.state.colonIndex + 1, this.state.selection.start)}
```